UI/js-src/lsmb/Form.js

```diff
@@ -20,10 +20,10 @@ define([
                       this.inherited(arguments);
 
                       // <button> tags get rewritten to <input type="submit" tags...
-                      query('input[type="submit"]', this.domNode)
+                      query("input[type=\"submit\"]", this.domNode)
```

Comment:
@ylavoie can't we use `query("input[type='submit']"` here then? much easier to read than all the escaped dquotes